logzio/.gitignore

```diff
@@ -0,0 +1 @@
+**/.DS_Store
```

Comment:
Can you fix the project's gitignore to ignore all DS_Store files instead of adding a new one here?

https://github.com/DataDog/integrations-extras/blob/master/.gitignore#L89

Reply:
Thanks for catching that. That `.gitignore` shouldn't be there